source/core/parameters.cc

```diff
@@ -1246,6 +1246,11 @@ namespace Parameters
         "false",
         Patterns::Bool(),
         "Bool to define if you assemble the inside of particles with the NS equation.");
+      prm.declare_entry(
+        "length ratio",
+        "4",
+        Patterns::Double(),
+        "The length ratio used to define the points for the IB stencil. See definition of epsilone_n in the paper on sharp IB.");
```

Comment:
```suggestion
        "The length ratio used to define the points for the IB stencil. See definition of epsilon_n in the paper on sharp IB.");
```

Hopefully this gets published soon.
Did you update wiki page?